### PR TITLE
Projects page

### PR DESCRIPTION
**Pull request by dwhswenson**
For your consideration. Not that we have to do it -- just something I thought might help us make our pitch:

 
image

### CODE REVIEW

**Comment by dwhswenson** [comment]
This can be reviewed. Nothing urgent here, but it directly links to our various repos from the main website, instead of only being findable if they click through via our org GitHub page. And even if people don't click through to the repos, it's a good overview of what we're working on.

**Comment by dwhswenson** [comment]
This can be reviewed. Nothing urgent here, but it directly links to our various repos from the main website, instead of only being findable if they click through via our org GitHub page. And even if people don't click through to the repos, it's a good overview of what we're working on.

File: _projects/gufe.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: gufe
```
Comment (IAlibay) [inline _projects/gufe.md:2]: @richardjgowers do we want to advertise gufe in this way? I was under the impression that it technically is an openfe component "for those in the know"

File: _projects/lomap.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+name: lomap
+repo: OpenFreeEnergy/Lomap
+conda-install: lomap2
+role: maintainers
+description: >-
+  Lomap is a tool for creating atom mappings between two ligand molecules.
+  It was originally developed by the research lab of David Mobley at
```
Comment (IAlibay) [inline _projects/lomap.md:8]: Can we have a link to the original paper and maybe a link to the defunct repo?

File: _projects/openfe.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+name: openfe
+repo: OpenFreeEnergy/openfe
+docs: https://openfe.readthedocs.org
+conda-install: openfe
+role: flagship
+description: >-
+  This is the flagship product of Open Free Energy. This library contains
+  tools to create, simulate, and analyze networks of alchemical
+  transformations. It also includes a command line interface and interactive
+  visualization tools.
```
Comment (IAlibay) [inline _projects/openfe.md:11]: Should we have a link to try.openfree.energy here too?

**Review by IAlibay — CHANGES_REQUESTED** [review]
Couple of small changes - mainly adding some extra info. If you think it's not worth being more verbose I'm happy to go that way too.

There's the one query with guefe too, we might not want to advertise it 😅 (or at least that way Richard's promise to some folks in order to have the name)

File: projects.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+permalink: /projects/
+description: >-
+  Open Free Energy develops and maintains several software tools used in
+  alchemical free energy calculations.
+---
+
+<table class="projects">
+  {% assign roles = "flagship developers maintainers" | split: " " %}
+  {% for role in roles %}
+  {% for project in site.projects %}
```
Comment (richardjgowers) [inline projects.html:12]: can we hack the jinja here to list openfe at the top of the list?

File: _projects/gufe.md (hunk 1 above)
Comment (richardjgowers) [reply _projects/gufe.md:2]: Ehhh not sure, I guess as long as it's not at the top of the list (see comment below)

Comment (richardjgowers) [reply _projects/gufe.md:2]: I was planning on documenting gufe as part of the openfe package, it's just split out for logistical reasons, but maybe we should dogfood it more than that

File: projects.html (hunk 4 above)
Comment (dwhswenson) [reply projects.html:12]: Already done, as long as openfe remains the only thing marked as a "flagship" project.

File: _projects/gufe.md (hunk 1 above)
Comment (dwhswenson) [reply _projects/gufe.md:2]: This is intended to be a "for the moment" list. In the future, we might remove gufe. Right now, my goal is for the projects page to basically say "see, we really are doing a lot of work!" to potential funders -- listing many projects makes us look a little more thoroughly established. (We might add the PL benchmarks repo here, too.) I'm thinking of it like a CV: when you're young and haven't done much, you put everything on there. Once you have a decent bit of experience, you only include the highlights.

File: _projects/lomap.md (hunk 2 above)
Comment (dwhswenson) [reply _projects/lomap.md:8]: For the defunct repo, adding a link in the Lomap repo README in https://github.com/OpenFreeEnergy/Lomap/pull/12 -- I don't think we need that here as well. I will add a link to the paper here; checking that I have the right reference (see  also https://github.com/OpenFreeEnergy/Lomap/pull/11).

Comment (dwhswenson) [reply _projects/lomap.md]: link to paper added

**Review by IAlibay — APPROVED** [review]
assuming we keep gufe, lgtm!

**Pull request merged**